assignment.py

```diff
@@ -0,0 +1,34 @@
+'''3. Write a program to create "BCA.txt" file which contain information about BCA course. 
+   count and print the total number of lines starting with ‘A’, ‘B’, and ‘C’ in "intro.txt" file.'''
+# Create the BCA.txt file with BCA course information
+bca_info = """
+BCA Course Information:
+
+- BCA stands for Bachelor of Computer Applications.
+- It is a 3-year undergraduate program in the field of computer applications.
+- The course covers various aspects of computer science and programming.
+- Graduates of BCA can pursue careers in software development, IT consulting, etc.
+"""
+
+with open("BCA.txt", "w") as f:
+    f.write(bca_info)
+
+# Count lines starting with 'A', 'B', and 'C' in intro.txt file
+file = "intro.txt"
+a = 0
+b = 0
+c = 0
+
+with open(file, "r") as d:
+    for line in d:
+        if line.strip().startswith('A'):

```

Comment:
what is the meaning of line.strip()? can you explained it.

Reply:
"line.strip()" removes any leading and trailing spaces, tabs, or newline characters from a string called line.